wordpress-mobile/WordPress-iOS
# Update stats in the iOS 14 today widget from the app

Fixes #15104

Using the existing Stats logic to update the new Today Widget when the stats in the app are updated.

Known issues:

- at the moment, the logic takes as "default" site whatever site has been set for the old widgets (using the "Widgets" button in the Stats scene). If none was set, this will be the default site for the current user
- at the moment the logout and re-login with a different user is not handled, so it should be tested with only one user

To test:
- run the app and login on the testing device with a user that has at least one site, and possibly some daily stats that are not all zero
- background the app, Install the widget (any size would do, but medium size visualizes more info) and verify that the info look correct
- trigger some stats updates (e.g. do some visits, likes, comments from another device or browser, with another user that is following the site you are testing)
- go back to the app, then choose the site that is shown in the widget and then go to Stats, making sure that the updates you triggered appear
- background the app and go to the widget, and make sure that the same updates are reported in the widget.


PR submission checklist:
